pkg/apis/experimental/v1alpha1/custom_builder_validation_test.go

```diff
@@ -22,9 +23,15 @@ func testCustomClusterBuilderValidation(t *testing.T, when spec.G, it spec.S) {
 		},
 		Spec: CustomNamespacedBuilderSpec{
 			CustomBuilderSpec: CustomBuilderSpec{
-				Tag:   "some-registry.io/custom-builder",
-				Stack: "some-stack",
-				Store: "some-registry.io/store",
+				Tag: "some-registry.io/custom-builder",
+				Stack: corev1.ObjectReference{
```

Comment:
Was the intention of enforcing specification of the "kind" of stack/store to support future extensibility (i.e.: in case there are other stack/store kinds developed in the future that the CustomBuilder accepts)?

Reply:
@piyalibanerjee yup!